Symfony/CS/Tokenizer/Token.php

```diff
@@ -268,6 +268,34 @@ public static function getKeywords()
     }
 
     /**
+     * Generate array containing all predefined constants that exists in PHP version in use.
+     *
+     * @see http://php.net/manual/en/language.constants.predefined.php
+     *
+     * @return array<int, int>
+     */
+    public static function getMagicConstants()
+    {
+        static $magicConstants = null;
+
+        if (null === $magicConstants) {
+            $magicConstants = array();
+            $magicConstantsStrings = array(
+                'T_CLASS_C', 'T_DIR', 'T_FILE', 'T_FUNC_C', 'T_LINE', 'T_METHOD_C', 'T_NS_C', 'T_TRAIT_C',
+            );
+
+            foreach ($magicConstantsStrings as $keywordName) {
```

Comment:
done, https://github.com/FriendsOfPHP/PHP-CS-Fixer/commit/62d0a345ba54c0b9767357083b047514d4734400
diff looks but odd...  should I move the private method to another place in the class?